docker/Dockerfile

```diff
@@ -86,6 +86,9 @@ RUN \
 	apt-get install -y git && \
 	echo "INFO: npm install --production..." && \
 	npm install --production && \
+	# Ensure permissions over default directory used by STH to generate csv files
+        mkdir temp && \
+        chmod 777 temp && \
```

Comment:
Indent is weird... maybe some [tab vs space](https://www.youtube.com/watch?v=SsoOG6ZeyUI) issue :)

Reply:
fixed in https://github.com/telefonicaid/fiware-sth-comet/pull/562/commits/6407409a8b529b1ad0691455913742d787f48853